episodes/web-location-identifiers.Rmd

```diff
@@ -0,0 +1,257 @@
+  <li>**Optional:** <i>Take a closer look on the different status codes or the HTTP responses.</i></li>
+  <li><i>Enforce an Error 404, e.g. by deleting one character in the file path. The status code 404 will appear in the status code column of the network editor.</i></li>
+</ul>
+
+<a href="../../docs/files/exampleError404.png" target="popup" onclick="window.open('../docs/files/exampleError404.png,'popup','width=1200'); return false;" onMouseOver="document.Error404.src='../../docs/files/CLICKexampleError404.png';" onMouseOut="document.Error404.src='../../docs/files/exampleError404.png';"><span><img src="files/exampleError404.png" alt="Screenshot of the recommended live coding session." name="Error404"/></span></a>
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+## Lecture: Persistent Identifier
+
+**Slide set**: <a href="../files/slideSet/PersistentIdentifier_CarpentriesIncubator_ScientificMetadata.pdf" target="_blank" download>PersistentIdentifier.pdf</a>
+
+**Time estimate**: XX min
+
+1. Simplified, there could be two reasons, why you end up with a 404 status code: Either you can't spell, or the resource path is broken.
+2. How do path's break.
+3. The URL you want to retrieve points to specific resources, stored on the host server. But within the life cycle of an online resource, several events can take place.
+4. The file path on the host server might change, the resource might be migrated to a different host server, or the resource might simply be deleted. In any of these cases, the URL does not specify the exact location of the resource any longer and will no longer be retrievable by the server adressed by the GET-request you send from your browser. This process is also referred to as "link rot". Remember, that we classified the URL as a form of Uniform Resource Identifier? This means, that link rot leads results in the resource loosing a very important identifier!
+5. Web technology has a solution for this issue: Persistent Identifiers or short, PIDs.
+6. Like our other persistent and unique identifiers, such as ISBN numbers, PIDs are long-lasting references to digital objects on the web. Any resource that has a representation on the web - articles, datasets, persons, instruments - can have a PID assigned. These PIDs remain constant over time, lead directly to the resource - so they also give information on the location and protocol - and, thus, ensure permanent identifiability, refencabilty and retrievability of online resources.
+7. The technology behind PIDs is fairly trivial. In PID databases, a PID is mapped to a URL, that retrieves a resource. Even though the URL changes, due to server migration or changes in the file path, the PID remains consistent.
```

Comment:
```suggestion
7. The technology behind PIDs is fairly simple. In PID databases, a PID is mapped to a URL, that retrieves a resource. Even though the URL changes, due to server migration or changes in the file path, the PID remains consistent.
```

I wouldn't use "trivial" synonymously for "simple" outside of jargon-laden scientific papers to sound high-brow, unless it really is trivial. To the learners, this definitely is not "trivial" like 1+1 = 2